app/controller/window/MinimizableWindow.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * This class is the controller for the MinimizableWindow.
+ */
+Ext.define('CpsiMapview.controller.window.MinimizableWindow', {
+    extend: 'Ext.app.ViewController',
+
+    alias: 'controller.cmv_minimizable_window',
+
+    /**
+     * Sets the window invisible and calls the addMinimizedWindow event
+     * of the related MinimizedWindows toolbar.
+     */
+    onMinimize: function () {
+        var me = this;
+        var minimizeTo = me.getMinimizeToolbar();
+        if (minimizeTo == null) {
+            Ext.log({
+                msg: 'No cmv_minimized_windows_toolbar found. Window might just disappear.',
+                level: 'warn'
+            });
+        }
+        me.getView().setVisible(false);
+        minimizeTo.fireEvent('addMinimizedWindow', me.getView());
```

Comment:
Should probably return the function after the warning? Otherwise minimizeTo is null here and throws an error

Reply:
Was fixed in https://github.com/compassinformatics/cpsi-mapview/pull/185